scale/job/execution/running/tasks/job_task.py

```diff
@@ -1,12 +1,12 @@
 """Defines the class for a job execution job task"""
 from __future__ import unicode_literals
 
-from job.execution.running.tasks.base_task import Task
+from job.execution.running.tasks.exe_task import JobExecutionTask
 from job.resources import NodeResources
 
 
-class JobTask(Task):
-    """Represents a job execution job task (runs the actual job/algorithm)
+class JobTask(JobExecutionTask):
```

Comment:
Yeah, I think just about any name that doesn't reuse an existing concept for a different purpose is better in this case. Something like main would be fine.